utils/build-llvm-local.sh

```diff
@@ -53,8 +59,17 @@ cmake ../llvm \
     -DLLVM_OPTIMIZED_TABLEGEN=OFF \
     -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
     -Dpybind11_DIR=${PYTHON_ROOT}/pybind11/share/cmake/pybind11 \
-    -DLLVM_DISTRIBUTION_COMPONENTS="cmake-exports;not;count;FileCheck;MLIRPythonModules;mlir-cpu-runner;mlir-linalg-ods-yaml-gen;mlir-opt;mlir-reduce;mlir-tblgen;mlir-translate;mlir-headers;mlir-cmake-exports" \
-    |& tee cmake.log
+    -DLLVM_DISTRIBUTION_COMPONENTS=cmake-exports;not;count;FileCheck;MLIRPythonModules;mlir-cpu-runner;mlir-linalg-ods-yaml-gen;mlir-opt;mlir-reduce;mlir-tblgen;mlir-translate;mlir-headers;mlir-cmake-exports"
```

Comment:
Just curious: what does this do?

Reply:
Oh cool it's a way to install FileCheck and stuff. Do you need all those others though?

Reply:
> Oh cool it's a way to install FileCheck and stuff. Do you need all those others though?

Those were inherited from the AIR build script and I haven't experimented by disabling some of them.